scripts/src/chartprreview/verify-report.sh

```diff
@@ -141,7 +141,7 @@ getAnnotations() {
 
 
   while IFS= read -r line; do
-    line=`echo $line | xargs`
+    line="$(echo -e "${line}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
```

Comment:
we are reading here yaml structure, and reading it is line by line is very fragile and can catch things on e.g. wrong level. We should either use tools like [yq](https://github.com/mikefarah/yq) or perhaps this script should be written in Python or some other language with better yaml handling?

Reply:
I am fine with converting this script to Python. For now, let's merge this PR.

@mmulholla Any thoughts?

Reply:
ok, if this unblock you for a moment, but let's create an issue addressing this.

Reply:
I have created an issue here: #74